sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample5_TrainModel.md

```diff
@@ -24,6 +24,7 @@ var client = new FormTrainingClient(new Uri(endpoint), credential);
 Train custom models to recognize all fields and values found in your custom forms. A `CustomFormModel` is returned indicating the form types the model will recognize, and the fields it will extract from each form type.
 
 ```C# Snippet:FormRecognizerSample4TrainModelWithForms
+// For this sample, you can use the training forms found in the `trainingFiles` folder.
```

Comment:
do you think it's worth adding a sentence about uploading the files to blob storage and creating the container sas url? I'm worried people will just try to use the local path.